xcodebuild/utils.go

```diff
@@ -0,0 +1,383 @@
+package xcodebuild
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"regexp"
+	"strconv"
+	"syscall"
+	"time"
+
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-io/go-utils/progress"
+	cache "github.com/bitrise-io/go-xcode/xcodecache"
+	"github.com/kballard/go-shellquote"
+)
+
+// XcprettyTool ...
+const XcprettyTool = "xcpretty"
```

Comment:
We already have a constant for this in the step, no?

Reply:
Output tool and test repetition-related consts moved to xcodebuild.